src/node/rpc/frontend.h

```diff
@@ -488,14 +484,20 @@ namespace ccf
     void set_root_on_proposals(
       const enclave::RpcContext& ctx, kv::CommittableTx& tx)
     {
-      if (ctx.get_request_path() == "/gov/proposals")
+      if (
```

Comment:
As per the comment below, we should only ever use this for one specific endpoint, so let's make sure we proof this for later in case we add a new `GET /gov/proposals`